packages/aws-cdk-lib/aws-synthetics/lib/runtime.ts

```diff
@@ -247,6 +247,14 @@ export class Runtime {
    */
   public static readonly SYNTHETICS_NODEJS_PUPPETEER_9_1 = new Runtime('syn-nodejs-puppeteer-9.1', RuntimeFamily.NODEJS);
 
+  /**
+   * `syn-nodejs-playwright-1.0` includes the following:
+   * - Lambda runtime Node.js 20.x
+   * - Playwright version 1.44.1
+   * - Chromium version 126.x
```

Comment:
I added more stuff in the "new features" section of [my PR](https://github.com/nmussy/aws-cdk/blob/ffc66038f9ad447eae152ea87f87c75eb56c20a3/packages/aws-cdk-lib/aws-synthetics/lib/runtime.ts#L250-L265) if you're interested, but at least these fields should be updated

```suggestion   
   * - Playwright version 1.45
   * - Chromium version 126.0.6478.126
   *
   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_nodejs_playwright.html#Synthetics_runtimeversion-syn-nodejs-playwright-1.0
```